### PR TITLE
chore(release): release 3.3.0

### PR DESCRIPTION
**Pull request by kikito**
- chore(scripts) restore release-lib.sh
- docs(COPYRIGHT): update copyright for 3.3.0
- docs(kong-admin-api.yml): update Admin API definition for 3.3.0
- release: 3.3.1

### Checklist

- [ ] The Pull Request has tests
- [ ] There's an entry in the CHANGELOG
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Full changelog

* [Implement ...]

### Issue reference

Fix #_[issue number]_

### CODE REVIEW

File: kong-3.0.1-0.rockspec [hunk 1]
```diff
@@ -1,10 +1,10 @@
 package = "kong"
-version = "3.3.0-0"
+version = "3.0.1-0"
```
Comment (locao) [inline kong-3.0.1-0.rockspec:2]: The filename and this change seem wrong.

File: kong-3.0.1-0.rockspec [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 rockspec_format = "3.0"
 supported_platforms = {"linux", "macosx"}
 source = {
   url = "git+https://github.com/Kong/kong.git",
-  tag = "3.3.0"
+  tag = "3.0.1"
```
Comment (locao) [inline kong-3.0.1-0.rockspec:7]: Same here.

File: kong/meta.lua [hunk 3]
```diff
@@ -1,7 +1,7 @@
 local version = setmetatable({
   major = 3,
-  minor = 3,
-  patch = 0,
+  minor = 0,
+  patch = 1,
```
Comment (locao) [inline kong/meta.lua:4]: Ditto.

**Review by locao — CHANGES_REQUESTED** [review]

File: kong-3.0.1-0.rockspec (hunk 1 above)
Comment (kikito) [reply kong-3.0.1-0.rockspec:2]: It is part of our changelist. After we release, we bump the version numbers. We do the same in master (when we made feature freeze, we bumped the version number in master to 3.4)

Comment (kikito) [reply kong-3.0.1-0.rockspec:2]: oh but you are right, it should be 3.3.1, not 3.0.1. I must have tripped over the keyboard, fixing.

Comment (kikito) [reply kong-3.0.1-0.rockspec]: Should be good now

**Review by locao — APPROVED** [review]

**Comment by outsinre** [comment]
Should this PR cherry-pick to EE side?

**Pull request merged**